### DISCUSSION TITLE
Claude Support

### DISCUSSION THREAD

**Discussion by saliksik** [ChatGPTNextWeb/NextChat#2813] [Ideas]
Do you have any plans on supporting Claude API?

Thank you.

**Comment by davidmoore-io** [comment]
+1 on this, particularly given the new 2.1 200k token model, which they've exposed via their API

Https://www.anthropic.com/index/claude-2-1

**Comment by H0llyW00dzZ** [comment]
not sure if its supported or not because I haven't figure out for other ai (don't have access for testing)
